framework/encode/vulkan_capture_manager.cpp

```diff
@@ -2711,9 +2711,17 @@ void VulkanCaptureManager::OverrideGetPhysicalDeviceSurfacePresentModesKHR(uint3
 {
     assert((pPresentModeCount != nullptr) && (pPresentModes != nullptr));
 
-    for (uint32_t i = 0; i < (*pPresentModeCount); ++i)
+    if (common_manager_->GetForceFifoPresentModeSetting())
     {
-        pPresentModes[i] = VK_PRESENT_MODE_FIFO_KHR;
+        GFXRECON_LOG_WARNING_ONCE(
+            "Force all present modes in vkGetPhysicalDeviceSurfacePresentModesKHR to VK_PRESENT_MODE_FIFO_KHR,"
+            "The application will use VK_PRESENT_MODE_FIFO_KHR present mode in vkCreateSwapchainKHR, which may"
```

Comment:
just a small remark. it looks like it's missing a space at the end of the line: "... which maycause ..."

Reply:
I have fixed it, thanks.